engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableFaceCulling.java

```diff
@@ -19,7 +19,10 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class enable OpenGL's Face Culling, i.e. to render only the inside faces of the skysphere.
+ * Enables OpenGL's Face Culling.
+ *
+ * This can be used to discard the fragments that face away from the camera, and can be used to improve
+ * the performance in certain situations like rendering only the inside faces of the skysphere.
```

Comment:
Hmmm... I see where you are aiming at. 

The problem I see is that "a particular direction" is very, very open. As in: the set of all unit vectors starting from the origin: it's a lot of them! Here we are dealing with directions that are orthonormal to a triangle. And there's only two of them possible in that context.

I can see how there might be reasons to discard triangles facing toward the camera, so perhaps my text was too narrow. See if you can find a wording that suggest what's the typical use (culling triangles facing away from the camera for optimization purposes) and what the alternative uses could be.